src/components/course/data/hooks.jsx

```diff
@@ -599,7 +604,7 @@ export const useUserSubsidyApplicableToCourse = () => {
       subscriptionLicense,
       containsContentItems,
       missingSubsidyAccessPolicyReason,
-      enterpriseOffers: currentEnterpriseOffers,
+      enterpriseOffers,
```

Comment:
[inform] Since this is the `getMissingApplicableSubsidyReason` inputs, it needs to consider _expired_ offers, not `currentEnterpriseOffers`.